src/init.cpp

```diff
@@ -405,7 +405,7 @@ void SetupServerArgs()
     gArgs.AddArg("-maxtimeadjustment", strprintf("Maximum allowed median peer time offset adjustment. Local perspective of time may be influenced by peers forward or backward by this amount. (default: %u seconds)", DEFAULT_MAX_TIME_ADJUSTMENT), false, OptionsCategory::CONNECTION);
     gArgs.AddArg("-maxuploadtarget=<n>", strprintf("Tries to keep outbound traffic under the given target (in MiB per 24h), 0 = no limit (default: %d)", DEFAULT_MAX_UPLOAD_TARGET), false, OptionsCategory::CONNECTION);
     gArgs.AddArg("-onion=<ip:port>", "Use separate SOCKS5 proxy to reach peers via Tor hidden services (default: -proxy)", false, OptionsCategory::CONNECTION);
-    gArgs.AddArg("-onlynet=<net>", "Only connect to nodes in network <net> (ipv4, ipv6 or onion)", false, OptionsCategory::CONNECTION);
+    gArgs.AddArg("-onlynet=<net>", "Actively connect to remote nodes exclusively in network <net> (ipv4, ipv6 or onion). Passive, incoming connections are not affected by this parameter.", false, OptionsCategory::CONNECTION);
```

Comment:
It may be helpful if the text includes "... only net..." to match the arg name `onlynet`. So how about this tweak?
> Actively connect to remote nodes <del>exclusively</del><ins>only</ins> in network....

Reply:
Or, never mind that, I guess. It may seem ambiguous, like the remote node must not have multiple network connections available to it.

Reply:
I think You could have a point here.
How about "only in network..." like it was before?
>Actively connect to remote nodes ~~exclusively~~only in network....

Allthough "exclusively" sounds more precise to me. Maybe English native speaker(I am not) could comment.

Reply:
To me (a native English speaker), "only through" is quite natural and unambiguously describes the network route, as opposed to the unintended alternative interpretation that "exclusively in" conveys where the user may expect the remote node to be isolated in a single network.

Reply:
Thanks for Your explanation. I will try to clone my github fork repo to local computer to rebase and squash the current and (Your suggested) followup commit, as described in [CONTRIBUTING.md](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md)